docs/contribute/app_services_upgrade.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Application Services version upgrades
+permalink: /contributing/app-services-upgrades
+---
+
+Upgrading to a newer [version of App Services][as_version] upgrades the dependencies for multiple components, so it can be prudent to run a quick smoke test before it lands to ensure new functionality works and there are no regressions in the existing components.
+
```

Comment:
Let's add a section about running `bors try`, as well.

This will run `test-ui-browser` task, which will execute on-device tests of a sample browser built against the A-S version. If it's totally broken, that should blow up.

Reply:
Sidenote, we need a `bors ASsmoketest` task of sorts, as well!